### PR TITLE
Add Entry: PHP String Function term hex2bin()

### LINKED ISSUE #1895: [Term Entry] PHP String Functions `hex2bin()`
### Reason/inspiration (optional)

We would like a new term entry for [PHP](https://www.codecademy.com/resources/docs/php) about the `hex2bin()` function.

The entry should go in a new file in docs/content/php/concepts/string-functions/terms/hex2bin/hex2bin.md

The entry should

- Have a syntax section illustrating the function with a ` ```pseudo ` code block.
- An example showing the function in use with a ` ```php ` code block.
- A runnable codebyte example in a ` ```codebyte/php` block.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by sriharshamajeti**
### Description

Add entry for PHP string functions and new term for it, `hex2bin()`.

Closes #1895

### Type of Change

- Adding a new entry

### Checklist

- [ x ] All writings are my own.
- [ x ] My entry follows the Codecademy Docs style guide.
- [ x ] My changes generate no new warnings.
- [ x ] I have performed a self-review of my own writing and code.
- [ x ] I have checked my entry and corrected any misspellings.
- [ x ] I have made corresponding changes to the documentation if needed.
- [ x ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ x ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ x ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/php/concepts/string-functions/terms/hex2bin/hex2bin.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+```codebyte/php
+<?php
+  $hex_string = '48656c6c6f20576f726c6421';
+  $binary_string = hex2bin($hex_string);
+  echo $binary_string . PHP_EOL;
+?>
+```
+
+**Note**:
```
Comment (SSwiniarski) [inline content/php/concepts/string-functions/terms/hex2bin/hex2bin.md:38]:
```suggestion
```

File: content/php/concepts/string-functions/terms/hex2bin/hex2bin.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+- This function does **NOT** convert a hexadecimal number to a binary number.
```
Comment (SSwiniarski) [inline content/php/concepts/string-functions/terms/hex2bin/hex2bin.md:40]:
```suggestion
> **Note:** This function does **NOT** convert a hexadecimal number to a binary number.
```

File: content/php/concepts/string-functions/terms/hex2bin/hex2bin.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+- The `hex2bin()` function is available only in PHP 5.4.0 and later versions.
```
Comment (SSwiniarski) [inline content/php/concepts/string-functions/terms/hex2bin/hex2bin.md:41]:
```suggestion
The `hex2bin()` function is available only in PHP 5.4.0 and later versions.
```

File: content/php/concepts/string-functions/terms/hex2bin/hex2bin.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+```codebyte/php
+<?php
+  $hex_string = '48656c6c6f20576f726c6421';
+  $binary_string = hex2bin($hex_string);
+  echo $binary_string . PHP_EOL;
+?>
+```
+
+**Note**:
+
```
Comment (SSwiniarski) [inline content/php/concepts/string-functions/terms/hex2bin/hex2bin.md:39]:
```suggestion
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@sriharshamajeti I did a first review and left a few comments, basically around formatting your "Note". In docs we format these with a blockquote and a bolded "Note:" on the same line, like this:

```
> **Note:** Text of note.
```

**Review by SSwiniarski — APPROVED** [review]
@sriharshamajeti looks good for a second review.

**Comment by sriharshamajeti** [comment]
Hi @SSwiniarski one of the tests is failing -> `format:verify` what is wrong with the PR, as the name suggests is there something I am missing in the format?

**Comment by sriharshamajeti** [comment]
Thank you @SSwiniarski , will sure keep in mind.

File: content/php/concepts/string-functions/terms/hex2bin/hex2bin.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+The **`hex2bin()`** function converts a string of hexadecimal values to its binary representation.
+
+## Syntax
+
+```pseudo
+hex2bin(hex_string);
+```
+
+The `hex_string` parameter represents the hexadecimal values to be converted.
```
Comment (KTom101) [inline content/php/concepts/string-functions/terms/hex2bin/hex2bin.md:24]:
```suggestion
The `hex_string` parameter represents the string of hexadecimal values to be converted.
```

**Review by KTom101 — CHANGES_REQUESTED** [review]
@sriharshamajeti I've added another review.

**Comment by SSwiniarski** [comment]
@sriharshamajeti In the interest of getting this merged, I implemented @KTom101's suggestions.

**Comment by SSwiniarski** [comment]
@KTom101, are we ready to merge?

**Comment by KTom101** [comment]
@sriharshamajeti So you know for future entries, please make a new feature branch for every entry. We discourage working off your `main` branch. 

In regards to the tests, I ran `npm run format:all` to fix the checks.

**Review by KTom101 — APPROVED** [review]

**Comment by KTom101** [comment]
@sriharshamajeti Link is live: https://www.codecademy.com/resources/docs/php/string-functions/hex2bin

image

Cc @yangc95

**Pull request merged**